neurow/lib/neurow/internal_api/endpoint.ex

```diff
@@ -9,6 +9,7 @@ defmodule Neurow.InternalApi.Endpoint do
   plug(MetricsPlugExporter)
 
   plug(Neurow.JwtAuthPlug,
+    auth_header_keys: ["x-interservice-authorization", "authorization"],
```

Comment:
📓 : the fallback is here

Reply:
Nit: Name this property `credential_headers`

Reply:
Done ✅